src/main/java/seedu/address/logic/commands/AddPersonCommand.java

```diff
@@ -60,6 +62,7 @@ public AddPersonCommand(Person person) {
         this.email = person.getEmail();
         this.internshipId = person.getInternshipId();
         this.tags.addAll(person.getTags());
+        this.linkIndex = person.getLinkIndex();
```

Comment:
Actually `Person` should not be keeping a `linkIndex` field since it should only be used in the command. We don't have to update this `AddPersonCommand` constructor to try to get a `linkIndex`, instead `this.linkIndex` should just be set to null here. This constructor is only used by the `Test` classes to  make `AddPersonCommand`s for predefined `Person`s.